spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -144,19 +150,13 @@
       it 'parses and stores the current level of assurance' do
         post :saml_callback, RelayState: fake_relay
         assert_response :found
-
-        session = Session.find(query_token)
-        user = User.find(session.uuid)
-        expect(user.loa[:current]).to eq(LOA::TWO)
+        expect(created_user.loa[:current]).to eq(LOA::TWO)
       end
 
       it 'parses and stores the highest level of assurance proofing' do
         post :saml_callback, RelayState: fake_relay
         assert_response :found
-
-        session = Session.find(query_token)
-        user = User.find(session.uuid)
-        expect(user.loa[:highest]).to eq(LOA::THREE)
+        expect(created_user.loa[:highest]).to eq(LOA::THREE)
```

Comment:
I couldnt resist fixing up old examples using my new lets - but this really isn't important for the PR.